Makefile

```diff
@@ -1,6 +1,103 @@
-VERSION := $(shell python3 setup.py --version)
-NEXT_VERSION := $(shell expr "$(VERSION)" + 1)
-COMMIT=$(shell git rev-parse HEAD)
+#
+# Maintenance Helpers
+#
+# This makefile contains targets used for development, as well as helpers to
+# aid automatization of maintenance. Unless a target is documented (or
+# mentioned elsewhere in official documentation), it is not supported and is
+# only meant to be used by developers to aid their daily development work.
+#
+# All supported targets honor the `SRCDIR` variable to find the source-tree.
+# For most unsupported targets, you are expected to have the source-tree as
+# your working directory. To specify a different source-tree, simply override
+# the variable via `SRCDIR=<path>` on the commandline. While you can also
+# override `BUILDDIR`, you are usually expected to have the build output
+# directory as working directory.
+#
+
+BUILDDIR ?= .
+SRCDIR ?= .
+
+RST2MAN ?= rst2man.py
```

Comment:
Should this be `rst2man` with the `.py` extension? That's what it was before and `python3-docutils` only contains that executable.

Reply:
Fixed.